<launch>

  <arg name="joystick_config" default="$(find scout)/config/joystick.json"/>
  <arg name="joystick_profile" default="DesktopXbox"/>
  <arg name="car_control_topic" default="/car/control/joystick"/>

  <node pkg="scout" type="joystick_node" name="scout_joystick" output="screen">
    <param name="joystick_config" value="$(arg joystick_config)" />
    <param name="joystick_profile" value="$(arg joystick_profile)" />
    <param name="car_control_topic" value="$(arg car_control_topic)" />
  </node>

  <node name="ds4_joystick" pkg="joy" type="joy_node">
    <param name="dev" value="/dev/input/js0" />
  </node>

</launch>

